components/Poll/PollVoteCard.tsx

```
@@ -0,0 +1,300 @@
+      {/* Poll Voting Card Header */}
+      <div className="flex justify-between items-center mb-3">
+        <div className="flex items-center gap-2">
+          <div className="w-9 h-9 rounded-full bg-gray-200 flex items-center justify-center">
+            <UserIcon />
+          </div>
+          <span className="text-sm text-gray-900">@mitch.1306</span>
+        </div>
+        <div className="flex items-center gap-1">
+          <div
+            className={`w-2 h-2 rounded-full ${
+              false ? "bg-red-500" : "bg-success-900"
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)